app/src/test/java/gov/va/vro/controller/ClaimMetricsControllerApiTest.java

```diff
@@ -0,0 +1,380 @@
+package gov.va.vro.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
This is the previous ClaimMetricsControllerTest renamed for clarity, with the last test moved out to the other file since mockito is not appropriate there.